zulip/zulip/__init__.py

```diff
@@ -718,6 +718,19 @@ def upload_file(self, file):
             files=[file]
         )
 
+    def get_attachments(self):
+        # type: () -> Dict[str, Any]
+        '''
+            Example usage:
+
+            >>> client.get_attachments()
+            {'result': 'success', 'msg': '', 'attachments':[{...}, {...}]}
```

Comment:
There should be a space after the `:` here.

Are you generating these manually rather than copy-pasting output?  That tends to result in typos like this.